TESTS/mbed_drivers/ticker/main.cpp

```diff
@@ -141,7 +142,7 @@ Case cases[] = {
 };
 
 utest::v1::status_t greentea_test_setup(const size_t number_of_cases) {
-    GREENTEA_SETUP((total_ticks + 5) * 3, "timing_drift_auto");
+    GREENTEA_SETUP((total_ticks + 5) * 3, "default_auto");
```

Comment:
This is an incorrect change, the host test should be set to `timing_drift_auto`. Please revert this.